Mathlib/Analysis/SpecialFunctions/Complex/Circle.lean

```diff
@@ -5,6 +5,7 @@ Authors: Yury G. Kudryashov
 -/
 import Mathlib.Analysis.Complex.Circle
 import Mathlib.Analysis.SpecialFunctions.Complex.Log
+import Mathlib.Algebra.Group.AddChar
```

Comment:
```suggestion
import Mathlib.Algebra.Group.AddChar
import Mathlib.Analysis.Complex.Circle
import Mathlib.Analysis.SpecialFunctions.Complex.Log
```

Reply:
This is a no-op, right? Or is the order of `import` statements significant somehow?

Reply:
We are semitrying to keep it alphabetical. And no it's not a no-op, although exceptions are hard to spot

Reply:
In case it wasn't clear, I have already applied your suggestion; thank you for explaining the reasoning behind it. Are you content with this PR now?

Reply:
Sorry, I'm reviewing every time I am commuting (the point of my github status being to let you know about this) and I have very little time on a computer with internet connection, so I can only do superficial reviews. Hopefully I will have time tonight to send this PR off, but no promise.